app/views/devise/registrations/edit.html.erb

```diff
@@ -35,12 +35,9 @@
             <%= f.button :submit, "Update", class: "btn btn-primary" %>
           </div>
         <% end %>
-
-        <h3>Cancel my account</h3>
-
+        <br>
+        <p><%= link_to "Cancel", :back, class: "btn btn-primary" %></p>
```

Comment:
I think it's supposed to be <a class="btn btn-outline-secondary my-3" href="/credit_cards">Cancel</a> *instead of btn-primary

Reply:
It's `btn btn-outline-secondary`